### ISSUE TITLE
Online Testbank: Which of the following are not globally based AWS services? (Choose two.)

### ISSUE THREAD

**Issue by tym0n** [awsccp/awsccp.github.io#59]
**Please describe the error you're reporting** Question code: tb235636.AWSCCPSG2E.c04.04

![image](https://github.com/awsccp/awsccp.github.io/assets/19637995/cf86126c-787a-4579-8b99-b67b13535a0b)

**Comment by dbclinton** [comment]
Yup. That one looks like the answer is backwards. Thanks for telling us.

**Comment by visma-sari-ollikainen** [comment]
This same issue is in the book chapter 4 review questions, so not only in the test bank.
